### ISSUE TITLE
Ads not blocked on Samsung UN40M5300, and some functionality limited

### ISSUE THREAD

**Issue by AbbieDoobie** [Perflyst/PiHoleBlocklist#92]
I am still seeing an ad on the home bar (bottom of the screen) with this blocklist enabled. Additionally some functionality has been limited, such as browsing the TV's app store. The icons are blocked (as expected, based on blocklist comment), but I am unable to download/update apps. All existing apps function normally.

Note that I have applied this using pfblockerng-devel, not pihole, but I've verified the list is in effect on my network. With the list disabled I can download from the app store normally.

**Comment by Perflyst** [comment]
Can you check your pihole logs which domains cause the mentioned issues?

**Comment by IstotaSwiatla** [comment]
Hi,
First of all, thank you for maintaining SmartTV.txt list. I do use it with pihole (via firebog).
One of the domains that is not blocked is cdn.samsungcloudsolution.com - you only have cdn.samsungcloudsolution.net on your list. Let me know if you want me to create a separate issue.
Thanks, Tom

**Comment by IstotaSwiatla** [comment]
I can see this domain also listed https://github.com/Perflyst/PiHoleBlocklist/issues/39

**Comment by IstotaSwiatla** [comment]
Created https://github.com/Perflyst/PiHoleBlocklist/pull/97

**Comment by Perflyst** [comment]
cdn.samsungcloudsolution.com was removed again because it is required for system update check on Samsung UE40F5500
